docs/user-guide/index.md

```diff
@@ -91,6 +91,26 @@ export const appReducer = createReducer(
 );
 ```
 
+If you would like to nest forms below the top level of the store use `onNgrxForm` with `createReducer` instead on your specific reducer:
+
+```ts
+import { createReducer } from '@ngrx/store';
+import { onNgrxForm } from 'ngrx-forms';
+
+export interface AppState {
+  featureX: FeatureX;
+  featureY: {
+    someOtherNestedSlice: SomeOtherNestedSlice,
+    myForm: FormGroupState<MyFormValue>;
+  }
+}
+
+export const myFormReducer = createReducer(
```

Comment:
This is really confusing. Are you trying to say that the form itself is a feature state?

Reply:
This just reflects the use case i had, maybe i'm misusing NGRX in some way? i have feature slices (featureX, featureY) with slices of state below, each with their own reducer.